clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -431,11 +438,11 @@ public void handle(SyncGroupResponse syncResponse,
                            RequestFuture<ByteBuffer> future) {
             Errors error = Errors.forCode(syncResponse.errorCode());
             if (error == Errors.NONE) {
-                log.info("Successfully joined group {} with generation {}", groupId, generation);
```

Comment:
Did we drop this message for some reason? Seems like a useful message (or refine it to explain sync group if necessary, or even move to debug it that's necessary). Or just reduce the log level it happens at?